src/Table.php

```diff
@@ -56,7 +56,7 @@ public function __construct(
         $this->queryFactory = $queryFactory;
         $this->tableEvents = $tableEvents;
         $this->rowClass = substr(static::CLASS, 0, -5) . 'Row';
-        if (count($this::PRIMARY_KEY) == 1) {
+        if (count($this::PRIMARY_KEY) < 2) {
             $this->primaryKey = new PrimarySimple($this::PRIMARY_KEY[0]);
```

Comment:
When I don't have a primary key, I get an "Undefined offset: 0" here.

Reply:
The problem here: missing test cases for these changes.